backend/src/modules/tournament-tree/core/domain/TournamentTree.ts

```diff
@@ -26,19 +28,25 @@ export class TournamentTree {
       tournamentTeams: props.tournamentTeams,
       entityIdGenerator: props.entityIdGenerator,
     });
+
+    const tournamentTreeArray = winnerTree.getTournamentTreeArray().map((item, index) => {
+      return { ...item, matchNumberInSequence: index + 1 };
+    });
+
     const tournamentTreeProps = {
       tournamentId: props.tournamentId,
-      tournamentTreeArray: winnerTree.getTournamentTreeArray(),
+      tournamentTreeArray: tournamentTreeArray,
       tournamentTeams: props.tournamentTeams,
     };
+
     return new TournamentTree(tournamentTreeProps);
   }
 
-  static setTournamentTreeFromDataBase(props: {
+  public static setTournamentTreeFromDataBase(props: {
```

Comment:
Nie powinno być takiej funkcji w domenie, bo nasza domena jest nieswiadoma tego gdzie my co zapisujemy. Mozesz zapisywac w bazie z Jsona itp. Albo w ogole nie zapisywac tylko trzymac w pamieci. 
Ale mozesz to na razie zostawic :) Przy okazji Wam pokaze jak z czyms takim sobie dac rade, bo nie jest to proste. 

Tylko nie wiem czemu znowu to `set` :)

Reply:
za szczegółowe CR.

Zamysłem tej funkcji było ustawienie drzewa z danych dostarczonych z zewnątrz, np z bazy dabych.
Chodziło o to aby nie tworzyć drzewa od początku i od początku zaczynać turniej, tylko aby wczytać drzewo  z turnieju w trakcie jego trwania.
Dzięki Twojemu komentarzowi wpadłem na nazwę typu:
`loadAlreadyExistingTree()`
Będzie progress?


Dzięki za szczegółowe CR

Reply:
@nowakprojects

Reply:
Już lepiej :) Ale coś Wam przy okazji pokażę.